src/components/Tool/FixEmailAddresses/FixEmailAddresses.test.tsx

```diff
@@ -115,28 +124,147 @@ describe('FixPhoneNumbers-Home', () => {
     expect(getByTestId('starIcon-testid2-0')).toBeInTheDocument();
   });
 
-  //TODO: Fix during MPDX-7946
-  it.skip('add an email address to second person', async () => {
-    const { getByTestId, getByDisplayValue } = render(<Components />);
-    await waitFor(() =>
-      expect(getByTestId('starIcon-testid2-0')).toBeInTheDocument(),
-    );
-    expect(getByTestId('textfield-testid2-0')).toBeInTheDocument();
-
-    const textfieldNew1 = getByTestId(
-      'addNewEmailInput-testid2',
-    ) as HTMLInputElement;
-    userEvent.type(textfieldNew1, 'email12345@gmail.com');
-    const addButton1 = getByTestId('addButton-testid2');
-    userEvent.click(addButton1);
-
-    expect(textfieldNew1.value).toBe('');
-    expect(getByTestId('textfield-testid2-2')).toBeInTheDocument();
-    expect(getByDisplayValue('email12345@gmail.com')).toBeInTheDocument();
+  describe('add email address', () => {
+    interface AddEmailAddressProps {
+      postSaveResponse: object;
+      emailAddressNodes: object[];
+      elementToWaitFor: string;
+      textFieldIndex: number;
+      addButtonId: string;
+      cache: InMemoryCache;
+    }
+
+    const addEmailAddress = async ({
```

Comment:
We shouldn.t have a component in a test like this. Ifn we want to test addEmailAddress we should import it rather than recreate it.